src/components/common/PrivateRoute.js

```diff
@@ -3,22 +3,28 @@ import { Route } from 'react-router-dom';
 import { withAuthenticationRequired } from '@auth0/auth0-react';
 import { connect } from 'react-redux';
 import * as actions from '../../state/actions/authTokenAction';
+import * as actions1 from '../../state/actions/profileAction';
```

Comment:
maybe this could be renamed from actions1 to something else more related to the type of action?